pepys_import/core/store/data_store.py

```diff
@@ -414,6 +414,10 @@ def add_to_platforms(
         return platform_obj
 
     def add_to_synonyms(self, table, name, entity, change_id):
+        # Blacklist certain tables, and don't Synonyms for them be created
+        if table in ["Sensors", "GeometrySubTypes"]:
```

Comment:
Can you use variables from `constants` here? I think we already import it.

Reply:
Thanks Baris - good catch. Fixed now.